### PR TITLE
clowderizing this job too

### PR DESCRIPTION
**Pull request by gburges**

### CODE REVIEW

File: .rhcicd/do-nothing.yaml [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
+      schedule: "0 0 * * *"
+      podSpec:
+        image: quay.io/app-sre/ubi8-ubi-minimal:8.6-902
+        command: 
+          - /bin/sh
+          - -c 
+          - exit 0
 parameters:
 - name: IMAGE_TAG
   value: ''
```
Comment (gwenneg) [inline .rhcicd/do-nothing.yaml:27]: The `do-nothing.yaml` file from notifications-backend contains a `JOBID` param with some kind of regex. Is it needed here?

Comment (gwenneg) [reply .rhcicd/do-nothing.yaml:27]: https://github.com/RedHatInsights/notifications-backend/blob/ffccfe8968a1fb4803783ea5bebcb2fee20591dd/.rhcicd/do-nothing.yaml#L29-L31

Comment (gburges) [reply .rhcicd/do-nothing.yaml:27]: true, i separated the names, so we don't need jobid anymore in either of them

**Review by gwenneg — APPROVED** [review]
LGTM

**Pull request merged**